### PR TITLE
DOCAPI-6422: EN docs for adaptive index granularity and some settings

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5628

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -377,52 +377,86 @@ By default: 1,000,000. It only works when reading from MergeTree engines.
...
-**Possible values**
+Possible values:
+
+Any positive integer.
+
+Default value: 163840.
+
+## merge_tree_min_bytes_for_concurrent_read {#setting-merge_tree_min_bytes_for_concurrent_read}
+
+If a number of bytes to read from one file of a [MergeTree*](../table_engines/mergetree.md)-engine table exceeds merge_tree_min_bytes_for_concurrent_read` then ClickHouse tries to perform a concurrent reading from this file on several threads.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md]: Dirty.

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -377,52 +377,86 @@ By default: 1,000,000. It only works when reading from MergeTree engines.
...
 
 Any positive integer.
 
-**Default value**: 1048576.
+Default value: 128 ✕ 8192.
+
+
+## merge_tree_max_bytes_to_use_cache {#setting-merge_tree_max_bytes_to_use_cache}
+
+If ClickHouse should read more than `merge_tree_max_bytes_to_use_cache` bytes in one query, it does not use the cash of uncompressed blocks. The [uncompressed_cache_size](../server_settings/settings.md#server-settings-uncompressed_cache_size) server setting defines the size of the cache of uncompressed blocks.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md]: cache, not cash!

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -377,52 +377,86 @@ By default: 1,000,000. It only works when reading from MergeTree engines.
...
 
 Any positive even integer.
 
-**Default value**: 8.
+Default value: 8.
 
 ## merge_tree_max_rows_to_use_cache {#setting-merge_tree_max_rows_to_use_cache}
 
-If ClickHouse should read more than `merge_tree_max_rows_to_use_cache` rows in one query, it does not use the cash of uncompressed blocks. The [uncompressed_cache_size](../server_settings/settings.md#server-settings-uncompressed_cache_size) server setting defines the size of the cache of uncompressed blocks.
+If ClickHouse should read more than `merge_tree_max_rows_to_use_cache` rows in one query, it does not use the cache of uncompressed blocks. The [uncompressed_cache_size](../server_settings/settings.md#server-settings-uncompressed_cache_size) server setting defines the size of the cache of uncompressed blocks.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md]: Motivation is not highlighted.

File: docs/en/operations/table_engines/mergetree.md [hunk 4]
```diff
@@ -126,17 +129,17 @@ MergeTree(EventDate, intHash32(UserID), (CounterID, EventDate, intHash32(UserID)
...
 
-For each data part, ClickHouse creates an index file that contains the primary key value for each index row ("mark"). Index row numbers are defined as `n * index_granularity`. The maximum value `n` is equal to the integer part of dividing the total number of rows by the `index_granularity`. For each column, the "marks" are also written for the same index rows as the primary key. These "marks" allow you to find the data directly in the columns.
+Each data part logically divided by granules. A granule is the smallest indivisible data set that ClickHouse reads when selecting data. ClickHouse doesn't split rows or values, so each granule always contains an integer number of rows. The first row of a granule is marked with the value of the primary key for this row. For each data part, ClickHouse creates an index file that stores the marks. For each column, whether it is in the primary key or not, ClickHouse also stores the same marks. These marks allow findind the data directly in the columns.
```
Comment (alexey-milovidov) [inline docs/en/operations/table_engines/mergetree.md]: Typo.

File: docs/en/operations/settings/settings.md (hunk 3 above)
Comment (BayoNet) [reply docs/en/operations/settings/settings.md]: I've added it in a string below.

**Pull request merged**